### PR TITLE
Improve bamparse

### PR DESCRIPTION
**Pull request by ccedmendoza**
- [X] Repair test problem 

- [X] Add sambamba filtering 

- [x] Add test to check the filtering (current tests run without filtering)

### CODE REVIEW

File: tools/bamparse/bamparse.xml [hunk 1]
```diff
@@ -44,32 +49,32 @@
...
-
-Sense, antisense or both sense and antisense alignments can be counted
-
-The library labels in the returned count table are taken from the input bam datasets
-names in the Galaxy history.
-    </help>
     <tests>
         <test>
-            <param name="input_list" value="alignment1.bam,alignment2.bam" ftype="tabular" />
+            <param name="input_list" value="alignment1.bam,alignment2.bam" ftype="bam" />
```
Comment (ccedmendoza) [inline tools/bamparse/bamparse.xml]: This was the test problem

**Comment by ccedmendoza** [comment]
[improve "Count alignments" Galaxy tool speed, as was done for small_rna_maps](https://trello.com/c/PDfSE14q/387-improve-count-alignments-galaxy-tool-speed-as-was-done-for-smallrnamaps)

File: tools/bamparse/test-data/table_antisense.tabular [hunk 2]
```diff
@@ -0,0 +1,239 @@
+gene	alignment1.bam	alignment2.bam
```
Comment (drosofff) [inline tools/bamparse/test-data/table_antisense.tabular]: This test seems weak ! (only 0, thus low complexity of the data, increased chance to pass the test for bad reasons)

**Comment by drosofff** [comment]
I think it's good 🎉 Nice job

**Pull request merged**